lib/handlers/quote/quote.ts

```diff
@@ -232,25 +233,25 @@ export class QuoteHandler extends APIGLambdaHandler<
     }
 
     let protocols: Protocol[] = []
+
+    const isMobileRequest = ['uniswap-ios', 'uniswap-android'].includes(params.event.headers['x-request-source'] ?? params.requestQueryParams.source ?? '');
+    const appVersion = params.event.headers['x-app-version'];
+    // We will exclude V2 if isMobile and the appVersion is not present or is lower or equal than 1.24
+    const excludeV2 = isMobileRequest && (appVersion === undefined || semver.lte(appVersion, '1.24'))
```

Comment:
I thought app version 1.24 is a good version, and should not be excluded. This might be important, because over half of install base was on 1.24 just a week ago, so worth triple checking.